clients/vscode/src/chat/dom.d.ts

```diff
@@ -0,0 +1,4 @@
+// We don't include DOM types in typescript building config
+
+// FIXME: This is required by `@quilted/threads` and `tabby-chat-panel`
+declare type HTMLIFrameElement = unknown;
```

Comment:
@wwayne  I remove DOM types dependencies in this refactor.  But `HTMLIFrameElement` is used by `@quilted/threads` and `tabby-chat-panel`.  Added this dummy declaration for tsc checks.